.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- **Package Name**: 
```

Comment:
Am going to yank the "package name" field for UAMQP since unlike azure-sdk there's only one choice.